app/trip/hooks/useTripUpdate.ts

```diff
@@ -71,6 +73,10 @@ export const useTripUpdate = (
 
       await Promise.all([...deletePromises, ...createPromises])
 
+      if (data.uploaded_image_file && tripDetails) {
+        await uploadFile(data.uploaded_image_file, tripDetails.id)
+      }
+
```

Comment:
After uploaded, it shows the form for a sec, and then redirect to details page. Could you fix it?

Reply:
@kanatagu  I didn't know the problem despite of researching 🤔
Could I skip it as another issue. 🙏

Reply:
@samuraikun 
Yes No problem!